types/koa/index.d.ts

```diff
@@ -464,17 +464,17 @@ declare class Application<
      * @param {string[]} [options.keys] Signed cookie keys
      * @param {boolean} [options.proxy] Trust proxy headers
      * @param {number} [options.subdomainOffset] Subdomain offset
-     * @param {boolean} [options.proxyIpHeader] proxy ip header, default to X-Forwarded-For
-     * @param {boolean} [options.maxIpsCount] max ips read from proxy ip header, default to 0 (means infinity)
```

Comment:
is this being consumed by a tool that doesn't understand typescript types? by default these redundant types should be removed.

Reply:
This is only jsdoc. I find it quite useful...